### DISCUSSION TITLE
how to customize layout file?

### DISCUSSION THREAD

**Discussion by magic-thomas** [filamentphp/filament#8156] [Help] [answered]
Hi . 

Where is the layout file. and how to customize it ?

**Comment by leandrocfe** [comment, marked as answer]
this `vendor/filament/filament/resources/views/components/layout` ?
What do you need to customize? [css](https://filamentphp.com/docs/3.x/panels/themes)?

**Reply by magic-thomas** [reply]
Somehow I want to rewrite layout page including css. 

I will check it.

Thanks.
